release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1785,6 +1785,10 @@ In the following tables, features are marked with the following statuses:
 
 * The DNF package manager included in {op-system-first} images cannot be used at runtime, because DNF relies on additional packages to access entitled nodes in a cluster that are under a Red Hat subscription. As a workaround, use the `rpm-ostree` command instead. (link:https://issues.redhat.com/browse/OCPBUGS-35247[*OCPBUGS-35247*])
 
+* Due to a change in storage account naming in {product-title} {product-version}, the Azure File Container Storage Interface (CSI) driver now alphabetically matches storage account of the Image Registry Operator. With this change, there is a known issue where the Azure File CSI driver fails to mount all volumes when the image registry is configured as private. The mount failures occur because the CSI driver tries to use the storage account of the Image Registry Operator, which is not configured to allow connections from worker subnets.
++
+As a temporary workaround, image registries should not be configured as private when using the Azure File CSI driver. This is a known issue and will be fixed in a future version of {product-title}. (link:https://issues.redhat.com/browse/OCPBUGS-42308[*OCPBUGS-42308*])
```

Comment:
s/image registries/image registry/g

Reply:
Hi @stevsmit @RomanBednar, IMO, we are better to add another workaround too, like https://issues.redhat.com/browse/OCPBUGS-38922 mentioned `using storageclass with networkEndpointType: privateEndpoint or specify another storage account` , in case that custom have to use private image registry.

Reply:
@xiuwang I did consider providing this workaround already and I strongly suggest we don't do that. CSI storage with private endpoints is untested for now and we can't tell for sure if that would not cause any issues in the future during upgrades for example. That's why we did not introduce any code to change the storage class in this way, so suggesting that as a workaround officially would have similar effect.

We might consider adding this as a feature in next releases but the priority is not assessed yet.

Reply:
@RomanBednar Thanks for the explaining, that makes sense. I didn't consider upgrade such situation.